wear/src/main/kotlin/au/com/shiftyjelly/pocketcasts/wear/ui/component/horologist/ProgressStateHolderStyled.kt

```diff
@@ -28,13 +28,13 @@ class ProgressStateHolderStyled(
     initial: Float,
     private val timestampProvider: TimestampProvider,
 ) {
-    private val actual = mutableStateOf(initial)
+    private val actual = mutableFloatStateOf(initial)
```

Comment:
This fixes the lint issue:

```
Hint: Prefer mutableFloatStateOf instead of mutableStateOf 
  [AutoboxingStateCreation from androidx.compose.runtime]
```